README.md

```diff
@@ -203,19 +203,19 @@ import { ItlyNode as Itly } from '@itly/sdk';
 
       load(options: PluginLoadOptions): void {...}
 
-      alias(userId: string, previousId?: string): void {...}
+      async alias(userId: string, previousId?: string): Promise<void> {...}
```

Comment:
@falconandy @ondrej I played around with this and found if we use `Promise<any>` instead `Promise<void>` the Plugin implementations can avoid the return type all together e.g. `async alias(userId: string, previousId?: string) {}`. Seems like a nicety to me. What do you think?

Reply:
@justin-iteratively Personally I don't like to use `any`. We can avoid `Promise<void>` in function signature if we write `return new Promise<void>((resolve, reject) => ...`